doc/source/contributing.rst

```diff
@@ -296,38 +280,44 @@ Some other important things to know about the docs:
 - The tutorials make heavy use of the `ipython directive
   <http://matplotlib.org/sampledoc/ipython_directive.html>`_ sphinx extension.
   This directive lets you put code in the documentation which will be run
-  during the doc build. For example:
-
-  ::
+  during the doc build. For example::
 
       .. ipython:: python
 
           x = 2
           x**3
 
-  will be rendered as
-
-  ::
+  will be rendered as::
 
       In [1]: x = 2
 
       In [2]: x**3
       Out[2]: 8
 
-  This means that almost all code examples in the docs are always run (and the
-  output saved) during the doc build. This way, they will always be up to date,
-  but it makes the doc building a bit more complex.
+  Almost all code examples in the docs are run (and the output saved) during the 
+  doc build. This approach means that code examples will always be up to date,
+  but it does make the doc building a bit more complex.
+
+.. note::
+
+    The ``.rst`` files are used to automatically generate Markdown and HTML versions
+    of the docs. For this reason, please do not edit ``CONTRIBUTING.md`` directly,
+    but instead make any changes to ``doc/source/contributing.rst``. Then, to
+    generate ``CONTRIBUTING.md``, use `pandoc<http://johnmacfarlane.net/pandoc/>`_
```

Comment:
Fixed! Didn't push a new `CONTRIBUTING.md` because the space gets eaten by the conversion anyway.